.github/workflows/pre-commit.yaml

```diff
@@ -10,24 +9,11 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Checkout main code
         uses: actions/checkout@v2
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
+      - name: Pre-Commit Action
```

Comment:
**[Nitpick, non-blocking]** I would call this something slightly more specific, like:

```suggestion
      - name: Run pre-commit checks
```